src/firebase/firebase.js

```diff
@@ -27,28 +29,99 @@ export const generateUserDocument = async (user, additionalData) => {
   if (!snapshot.exists) {
     var { email, displayName, photoURL } = user
     try {
-      await userRef.set({
-        displayName,
-        email,
-        photoURL,
-        ...additionalData,
-      })
+      await userRef.set({ displayName, email, photoURL, ...additionalData })
     } catch (error) {
       console.error("Error creating user document", error)
     }
   }
   return getUserDocument(user.uid)
 }
 
+export const getCurrentUser = () => {
+  console.log('the current user is', firebase.auth().currentUser)
+  return firebase.auth().currentUser
+
+  
+  
+}
+
+export const verifyEmail = (email, page) => {
+  // directions for how to send the email verification
+  const actionCodeSettings = {url: "https://10.0.1.26:8000/" + page,
```

Comment:
For all URLs, define an environments file. Stop hardcoding, because this isn't going to work once you deploy.